### DISCUSSION TITLE
Legend under stats column

### DISCUSSION THREAD

**Discussion by tismofied** [henrygd/beszel#188] [Ideas]
Would it be possible to add legend under the stats kinda like grafana does?
[Grafana example](https://grafana.com/docs/grafana/latest/panels-visualizations/configure-legend/)

**Comment by henrygd** [comment]
I can add something like the example below.

The problem is the docker charts. Past a certain number of containers it will take up too much space and become fairly useless.

We could conditionally display the legend if the number of containers is below a certain threshold. Or maybe keep it to one line but have it scroll horizontally.

![image](https://github.com/user-attachments/assets/e2139592-c55b-4ff3-a021-c6ccbf9d7dfe)

**Reply by tismofied** [reply]
The examples you provided are perfect for me lol.
